utils/lit/lit/formats/taef.py

```diff
@@ -64,10 +64,12 @@ def getTaefTests(self, dll_path, litConfig, localConfig):
 
         if litConfig.debug:
             litConfig.note('searching taef test in %r' % dll_path)
+        
+        select_arg = str.format('/select:"{}"', self.select_filter)
 
+        cmd = [self.te, dll_path, "/list", select_arg]
         try:
-            lines = lit.util.capture([self.te, dll_path, '/list'],
-                                     env=localConfig.environment)
+            lines,_,_ = executeCommandForTaef(cmd)
```

Comment:
Using executeCommandForTaef here because lit.util.capture has issues with quotes in the select parameter.